main.go

```diff
@@ -45,11 +45,40 @@ type buildOptions struct {
 	makeQEMU   bool
 }
 
+// List of repos for which the integration pipeline shall be run
+// It can be overridden with env. variable WATCH_REPOS_PIPELINE
+var defaultWatchRepositoriesPipeline = []string{
```

Comment:
How about we just drop this, and only rely on the environment variable? :thinking: 
Do we even need this fallback?

Reply:
Yes, we are not setting the env variable in our Mender deployment, so we need the fallback.

We could update the env variable every time we have a new repo, but I find the code is easier to track for us developers than to go into an env variable in the SRE side of things.

Reply:
Makes sense